imap_processing/tests/idex/test_idex_l2a.py

```diff
@@ -165,3 +168,77 @@ def test_calculate_snr_warning(caplog):
         "Unable to find baseline noise" in message
         for message in caplog.text.splitlines()
     )
+
+
+@pytest.mark.filterwarnings("ignore:invalid")
+@pytest.mark.filterwarnings("ignore:overflow")
```

Comment:
In our normal processing are we going to get these invalid/overflows potentially? Do we want to ignore them internal to the functions (rather than just in the tests) using something like: 

```python
with np.errstate(invalid="ignore", overflow="ignore"):
    my_potential_overflow_calculation()
```